pd-configuration-file.md

```diff
@@ -285,6 +285,11 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
 + 是否使用 Joint Consensus 进行副本调度。关闭该特性时，PD 将采用一次调度一个副本的方式进行调度。
 + 默认值：true
 
+### `enable-diagnosis` <span class="version-mark">从 v6.3 版本开始引入</span>
+
++ 是否开启诊断功能。开启特性时，PD 将会记录调度中的一些状态来帮助诊断。开启时会略微影响调度速度，在 Store 数量较多时会消耗较大内存。
++ 默认值：false
+
```

Comment:
```suggestion
### `enable-diagnosis` <span class="version-mark">从 v6.3.0 版本开始引入</span>

+ 是否开启诊断功能。开启特性时，PD 将会记录调度中的一些状态来帮助诊断。开启时会略微影响调度速度，在 Store 数量较多时会消耗较大内存。
+ 默认值：false

```

Reply:
Should it be diagnosis or diagnostic? @Oreoxmt

Reply:
> Should it be diagnosis or diagnostic? @Oreoxmt

IMO, it should be diagnostics. This configuration item provides details about scheduler status, which helps user to diagnosis.